src/date-picker.vue

```diff
@@ -153,6 +157,20 @@ export default {
           year: 'YYYY',
           month: 'YYYY-MM',
           time: 'HH:mm:ss',
+          duration: {
+            // Date to String
+            stringify: date => {
+              const today = new Date();
+              return date
```

Comment:
@Xanngo please add newline

Reply:
Following initial repo coding styles, as discussed in slack.